src/EventStore/EventStore.php

```diff
@@ -158,7 +157,7 @@ public function writeToStream($stream_name, WritableToStream $events, $expectedV
             )
         ;
 
-        $request->setHeader('ES-ExpectedVersion', intval($expectedVersion));
+        $request->setHeader('ES-ExpectedVersion', intval($expected_version));
```

Comment:
Not so important to me. I used underscores to highlight method parameters, but I'm not religious on it.